### ISSUE TITLE
Documentation translation (idea for 2022)

### ISSUE THREAD

**Issue by hturner** [rstats-gsod/gsod2021#1]
Suggested by @tdhock in [ R Contribution Working Group meeting](https://github.com/forwards/rcontribution/blob/master/team_minutes/2021-03-12.md), consider a translation project for 2022, e.g. of R manuals or documentation.

**Comment by SangeetM** [comment]
Excerpt from [R Contribution Working Group meeting](https://github.com/forwards/rcontribution/blob/master/team_minutes/2021-03-12.md) 
"TH: Would translations fit into this? e.g. of R manuals/documentation"
Did @tdhock mean to say translation of the manuals/docs into other languages (apart from default English), similar to what is found [here](https://cran.r-project.org/other-docs.html#nenglish)?

**Comment by hturner** [comment]
Yes, although as noted at the top of https://cran.r-project.org/other-docs.html, that page is no longer maintained. So an initial task (before making the proposal) would be to review which documentation is actively maintained/what other translations are available. Then we would need to consider which documentation is most important to translate into which language(s) to have the most impact.

**Comment by hturner** [comment]
Noting here as something to bear in mind for project: in the past, there have been issues with spam links (and worse) added to the otherwise well-translated information. This is one reason why the R project has found it hard to maintain a directory of translated material.

**Comment by tdhock** [comment]
I would think the introductory guides may have the most impact for beginners to R who do not read English.
More advanced material such as Writing R Extensions may be useful. 
But google translate gets 95% of the job done, so maybe it is not so worthwhile to spend GSOD effort on translation?
e.g. Google translate of one paragraph from Writing R Extensions into French is actually pretty comprehensible:

1.2.4 Utilisation du code C ++
R peut être construit sans compilateur C ++ bien qu'il en existe un (mais pas nécessairement installé) sur toutes les plates-formes R connues. À partir de la version R 4.0.0, un compilateur C ++ ne sera sélectionné que s’il est conforme à la norme 2011 («C ++ 11»). Une mise à jour mineure41 («C ++ 14») a été publiée en décembre 2014 et sera utilisée par défaut à partir de la version R 4.1.0 si elle est prise en charge. D'autres révisions «C ++ 17» (en décembre 2017) et «C ++ 20» (avec de nombreuses nouvelles fonctionnalités en décembre 2020) ont été publiées depuis.
